Mlem/Views/Tabs/Settings/Components/Views/Appearance/TabBar/TabBarSettingsView.swift

```diff
@@ -54,8 +55,16 @@ struct TabBarSettingsView: View {
                                        settingName: "Show Unread Count",
                                        isTicked: $showInboxUnreadBadge)
             }
+            
+            Section {
+                SwitchableSettingsItem(settingPictureSystemName: "paintpalette",
+                                       settingName: "Translucent",
+                                       isTicked: $isTranslucentBar)
+            }
         }
```

Comment:
This should be in Accessibility settings under a section labeled "Transparency," should be labeled "Opaque Insets," and should be off by default (this will also require switching that toggle down through the rest of the code, sorry)

Reply:
I adjusted it. "translucent insets" turned on would mean a translucent tab bar, did I get that right?

Reply:
Yep! Generally speaking our settings follow "default = on", although that's not particularly consistently enforced.